zammad/Chart.yaml

```diff
@@ -16,17 +16,21 @@ maintainers:
 dependencies:
   - name: elasticsearch
     repository: https://charts.bitnami.com/bitnami
-    version: 19.10.3
+    version: 19.17.0
     condition: zammadConfig.elasticsearch.enabled
   - name: memcached
-    version: 6.5.4
+    version: 6.9.0
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.memcached.enabled
   - name: postgresql
-    version: 12.6.4
+    version: 12.12.10
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.postgresql.enabled
   - name: redis
-    version: 17.11.7
+    version: 17.17.1
     repository: https://charts.bitnami.com/bitnami
     condition: zammadConfig.redis.enabled
+  - name: minio
+    version: 13.2.0
+    repository: https://charts.bitnami.com/bitnami
+    condition: zammadConfig.minio.enabled
```

Comment:
dow we need this by default as long var is not using it?
imho by default zammad should use the postgres db so this would not be needed?

Reply:
It is not required by default, that is true. I can disable it by default for now, but I need to think about it if that's really what we want for production systems.

Reply:
Turned off by 6205f2249c6ecfffe6947be13fd23eeb294ee7c6.